src/components/queryExpressionEditor/QueryExpressionEditor.scss

```diff
@@ -0,0 +1,24 @@
+.container {
```

Comment:
you can use `.overflow-auto` instead

Reply:
https://github.com/coveo/vapor/blob/0696ab8feb29ac04e5846cfb0b35ee2bc31b7fee/scss/utility/layout.scss